server.py

```diff
@@ -228,3 +231,17 @@ def semesterIDType(reqstr):
 
   # Convert Officer objects back to dictionaries before returning
   return jsonify([x.__dict__ for x in officers])
+
+calurl = 'https://calendar.google.com/calendar/ical/ca149os3pmnh0dcopr1jn2negg%40group.calendar.google.com/public/basic.ics'
+dtstrformat = '%Y-%m-%dT%H:%M:%S.%fZ'
```

Comment:
does this need to be defined outside of the function or should we move it inside the function?

Reply:
I'll put both of those lines inside the function. Dunno why I put it outside tbh.